dev/bots/test.dart

```diff
@@ -544,7 +544,7 @@ Future<void> _runAddToAppLifeCycleTests() async {
 
 Future<void> _runFrameworkTests() async {
   final bq.BigqueryApi bigqueryApi = await _getBigqueryApi();
-  final List<String> nullSafetyOptions = <String>['--enable-experiment=non-nullable', '--no-sound-null-safety'];
+  final List<String> nullSafetyOptions = <String>['--enable-experiment=non-nullable'];
```

Comment:
these tests now set the right version when the experiment is enabled but the library is not opted in